src/components/chips/js/chipsController.js

```diff
@@ -99,11 +99,23 @@ function MdChipsCtrl ($scope, $mdConstant, $log, $element, $timeout) {
   this.useOnRemove = false;
 
   /**
-   * Whether to use the onSelect expression to notify the component's user
-   * after selecting a chip from the list.
-   * @type {boolean}
+   * Watch for Select Changes if expressions are present
    */
-  this.useOnSelect = false;
+  if (this.onChipSelect || this.onChipDeselect) {
+    var self = this;
+
+    $scope.$watch(function() {
```

Comment:
I think you might be able to do this as `$mdChipsCtrl.selectedChip` which will increase performance since it won't be executing the function on every digest loop.